pkg/vcs/git.go

```diff
@@ -672,3 +567,142 @@ func ParseGitDiff(patch []byte) []string {
 	}
 	return files
 }
+
+type GitWrapper struct {
+	Dir      string
+	Precious bool
+	Sandbox  bool
+	Env      []string
```

Comment:
Can we unexport some of these fields if they are used only within this package?

Reply:
I've unexported `Precious`, the others are used outside (at least so far).